<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MakeTest</class>
 <widget class="QDialog" name="MakeTest">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1153</width>
    <height>672</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton_add">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>540</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>14</pointsize>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Ajoutez</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_exit">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>540</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>14</pointsize>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Quitter</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>130</y>
     <width>611</width>
     <height>241</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>569</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Reponses:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_r1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>569</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_r2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>569</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_r3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>569</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_r4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>569</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_r5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>569</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>10</y>
     <width>107</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="Date">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>9</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>DATE:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="Date_ed">
      <property name="font">
       <font>
        <family>Arial</family>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>10</y>
     <width>181</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="clock">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>9</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Heure :</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="digital_clock">
      <property name="font">
       <font>
        <family>Arial</family>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>460</y>
     <width>811</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>18</pointsize>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>chaque questions doit contenir au moins 2 reponses.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>420</y>
     <width>1041</width>
     <height>34</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>16</pointsize>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Exemple: &quot;-&gt; option&quot;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>661</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>18</pointsize>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>NB: toujours ajoutez &quot;-&gt; &quot; avant les propositions correctes.</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_q">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>611</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>499</width>
     <height>46</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Questions:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
